README.md

```diff
@@ -53,6 +53,26 @@ Since the `cdk.json` file is generally committed to source control, it should ge
     },
 ```
 - `manifest_fetch_url` (str) - The concurrency to use when retrieving IIIF manifests from your API. If not provided, the default will be used (2).
+- `amplify.auth.secret_name` (str) - The name of the AWS Secrets Manager secret used for authentication. **If no name is provided, then the frontend will not be password protected**
+- `amplify.auth.username_key` (str) - The key for the username field within the secret (default: "username").
+- `amplify.auth.password_key` (str) - The key for the password field within the secret (default: "password").
+
+```bash
+# no auth
+cdk deploy
```

Comment:
Let's make auth "on" by default. Maybe do something like `cdk deploy -c amplify.auth.secret_name=NO_AUTH` - and have it not deploy w/out either a secret or explicitly stating that. The reason we are doing it is because of potential unintended costs and if we have multiple dev instances, etc, people are going to forget.... We don't need to be too concerned about that cluttering the app when it's actually deployable for others since this auth solution is short term anyways until we move to Cognito.